moonshot/integrations/cli/redteam/session.py

```diff
@@ -260,18 +561,36 @@ def run_attack_module(args):
         attack_module_id = args.attack_module_id
         prompt = args.prompt
         system_prompt = args.system_prompt if args.system_prompt else ""
-        context_strategy = args.context_strategy or []
-        prompt_template = [args.prompt_template] if args.prompt_template else []
-        metric = [args.metric] if args.metric else []
-        optional_arguments = (
-            literal_eval(args.optional_args) if args.optional_args else {}
+        # context strategy and prompt template should come from the session instead of the command
+
+        prompt_template = (
```

Comment:
The logic here a bit complex, can break down the if else instead of lumping together

Reply:
split the codes into multiple if and else blocks